resources/migrations/000_migrations.yaml

```diff
@@ -11293,6 +11293,26 @@ databaseChangeLog:
                   name: remark
                   type: ${text.type}
                   remarks: 'Optional remarks explaining why these changes were made.'
+  - changeSet:
+      id: v43.00-047
+      author: qnkhuat
+      comment: Added 0.43.0. Grant permissions to create/edit subscription to all existing groups except Admin
+      changes:
+        - sql:
+            sql: >-
+              INSERT INTO permissions (group_id, object)
+              SELECT
+                all_users_group.id AS group_id,
+                '/general/subscription/' AS object
+              FROM (
+                SELECT id
+                FROM permissions_group
+                WHERE name <> 'Administrators'
+              ) all_users_group
```

Comment:
You're selecting any group that's not the admin group, and you're calling it `all_users_group`... that's **super** confusing and misleading. Name it `groups` or `non_admin_groups` or something like that